projects/plugins/crm/src/automation/class-automation-bootstrap.php

```diff
@@ -106,6 +106,7 @@ protected function register_triggers(): void {
 			\Automattic\Jetpack\CRM\Automation\Triggers\Quote_Updated::class,
 			\Automattic\Jetpack\CRM\Automation\Triggers\Transaction_Created::class,
 			\Automattic\Jetpack\CRM\Automation\Triggers\Transaction_Updated::class,
+			\Automattic\Jetpack\CRM\Automation\Triggers\ClientWPUser_Updated::class,
```

Comment:
I suspect this was the intent:

```suggestion
			\Automattic\Jetpack\CRM\Automation\Triggers\ClientWPUser_Created::class,
```